pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "unredact"
+version = "0.1.0"
+description = "This repo contains a script that removes weak redactions from PDFs."
+authors = ["Kyle Dent <kdent@seaglass.com>"]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.scripts]
+lint = "black && isort && ruff"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+black = "^23.7.0"
+ruff = "^0.0.281"
+isort = "^5.12.0"
+pdfminer = "^20191125"
+pillow = "^10.0.0"
+reportlab = "^4.0.4"
```

Comment:
To add dependencies to your package, you should be good to now tho, you use the command: `poetry add 'package'`.